src/app/requirements.txt

```diff
@@ -16,6 +16,8 @@ backports-zoneinfo==0.2.1
     #   tzlocal
 brewtils==3.14.0
     # via beer-garden (setup.py)
+bson==0.5.10
```

Comment:
I was confused by this one.  We definitely have direct imports of bson, but the fact that this is an addition to the requirement.txt and not a change indicates that it wasn't actually being installed before.  I dug into it a bit and it looks like the bson that we are using was coming from pymongo.  The [pymongo pypi page](https://pypi.org/project/pymongo/) specifically has this warning:

> Do not install the “bson” package from pypi. PyMongo comes with its own bson package; doing “easy_install bson” installs a third-party package that is incompatible with PyMongo.

What that means for this PR is that I think we want to actually take bson out here and then figure out what to do with pymongo as part of #1329.  It may be the case that our use of bson means we have the direct pymongo dependency after all, even though that relationship isn't obvious.